includes/class-coblocks-plugin-deactivation.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * Register plugin deactivation class.
+ *
+ * @package CoBlocks
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * CoBlocks_Plugin_Deactivation Class
+ *
+ * @since NEXT
+ */
+class CoBlocks_Plugin_Deactivation {
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+
+		add_action( 'admin_footer-plugins.php', array( $this, 'admin_coblocks_deactivation_modal' ) );
+
+		add_filter( 'admin_enqueue_scripts', array( $this, 'equeue_scripts' ) );
+
+	}
+
+	/**
+	 * Loads the asset file for the given script or style.
+	 * Returns a default if the asset file is not found.
+	 *
+	 * @param string $filepath The name of the file without the extension.
+	 *
+	 * @return array The asset file contents.
+	 */
+	public function get_asset_file( $filepath ) {
+
+		$asset_path = COBLOCKS_PLUGIN_DIR . $filepath . '.asset.php';
+
+		return file_exists( $asset_path )
+			? include $asset_path
+			: array(
+				'dependencies' => array(),
+				'version'      => COBLOCKS_VERSION,
+			);
+
+	}
+
+	/**
+	 * Register meta.
+	 */
+	public function equeue_scripts() {
+
+		global $pagenow;
+
+		if ( 'plugins.php' !== $pagenow ) {
```

Comment:
I would remove the `global $pagenow` in favor of using the `$hook_suffix` arg. See https://core.trac.wordpress.org/browser/tags/5.9/src/wp-admin/admin-header.php#L118